pages/sessions.jsx

```diff
@@ -232,7 +227,7 @@ var SpacesInfo = [
 
 var SessionsPage = React.createClass({
   spaces: SpacesInfo,
-  componentWillMount: function() {
+  render: function() {
```

Comment:
Don't worry about changing this. I wasn't clear on what it was doing, but now that I figured it out, it makes sense. (The className functionality could be done in CSS instead, but that's a different issue)